armi/cases/tests/test_cases.py

```diff
@@ -269,12 +269,7 @@ def test_clone(self):
 
 
 class TestCaseSuiteDependencies(unittest.TestCase):
-    """CaseSuite tests.
-
-    .. test:: Dependence allows for one case to start after the completion of another.
-        :id: T_ARMI_CASE_SUITE
-        :tests: R_ARMI_CASE_SUITE
```

Comment:
Did this test tag just disappear?

I think you can move it down to `test_explicitDepdendency()`.